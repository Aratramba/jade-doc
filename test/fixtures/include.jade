//- @jadedoc
  name: inclusion tag
  description: this is an inclusion tag

div
  include tag.jade

//- @jadedoc
  name: inclusion tag
  description: this is an inclusion tag

div
  include mixins.jade
  +mixin1