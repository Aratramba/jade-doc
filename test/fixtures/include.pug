//- @pugdoc
  name: inclusion tag
  description: this is an inclusion tag

div
  include mixins.pug
  include deprecated.jade
  +mixin2('foo', 'faa')