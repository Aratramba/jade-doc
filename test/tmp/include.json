[{"meta":{"name":"inclusion tag","description":"this is an inclusion tag"},"file":"test/fixtures/include.jade","source":"div\n  include mixins.jade\n  +mixin2('foo', 'faa')","output":"<div><div>this is a mixin foo</div><div>this is the same mixin faa</div></div>"}]