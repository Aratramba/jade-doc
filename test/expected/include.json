{
  "inclusion tag": {
    "jade": {
      "comment": "    name: inclusion tag\n    description: this is an inclusion tag",
      "type": "include",
      "indent": 2,
      "root": "  include 'node'",
      "code": "  include 'node'\n",
      "file": "./test/fixtures/include.jade"
    },
    "name": "inclusion tag",
    "description": "this is an inclusion tag",
    "html": ""
  }
}